app/routes/static_pages.js

```diff
@@ -0,0 +1,24 @@
+import accountDetails from "./account_details";
+
+export default accountDetails.extend({
+  setupController(controller, model, transition) {
+    this._super(...arguments);
+    let applicationController = this.controllerFor("application");
+    applicationController.set("showSidebar", true);
+    let pageTitle =
```

Comment:
this doesn't feel quite right. is there a way to pass in the pageTitle or use the targetName inside the i18n get...

```
this.get("i18n").t(transition.targetName + ".title")
```

Is that safe? 
cc @patrixr

Reply:
yeah, sure, I'll try this out, and change it. Its even better 👍 
Thanks @steveyken .

Reply:
```
this.get("i18n").t(`${transition.targetName}.title`)
```

I think it's fine to rely on translation keys, as long as it works seamlessly when we add new sections in the translation file. Doing `if/else` means that every time we add something in the file we need to come back here.